app/mailers/rent_mailer.rb

```diff
@@ -1,8 +1,8 @@
 class RentMailer < ApplicationMailer
-  def success_rent_email(user, rent, book)
-    @user = user
-    @rent = rent
-    @book = book
-    mail(to: user.email, subject: 'Awesome book rented')
+  def success_rent_email(user_id, rent_id)
+    @user = User.find(user_id)
+    @rent = Rent.find(rent_id)
```

Comment:
you can just pass the rent_id and obtain the user from it, one less parameter to save ;)